src/jobs/generation/Utilities.groovy

```diff
@@ -695,6 +698,9 @@ class Utilities {
      * @param defaultBranch Branch to build by default if this is NOT a PR job. Defaults to *&#x2215;master.
      */
     def static addStandardParameters(def job, String project, boolean isPR, String defaultBranch = '*/master') {
+		if (defaultBranch.indexOf('*/') == 0){
```

Comment:
Looking at the code in dotnet's org, can you remove the default value from defaultBranch?

Reply:
Nit: indentation

Reply:
Can you make this method private too?